ibis/backends/conftest.py

```diff
@@ -144,6 +144,12 @@ def is_older_than(module_name, given_version):
             "total_eva_hrs": "float64",
         }
     ),
+    "geo_wkb": ibis.schema(
+        {
+            "name": "string",
+            "geom": "binary",
+        }
+    ),
```

Comment:
entries here are also expected to have a `csv` file in testing-data -- might be better off sticking this in the duckdb `conftest.py` for now.